ext/pdo/pdo_dbh.c

```diff
@@ -1027,8 +1044,6 @@ PHP_METHOD(PDO, errorCode)
 
 	ZEND_PARSE_PARAMETERS_NONE();
 
-	PDO_CONSTRUCT_CHECK;
```

Comment:
question: why did you remove/not add the construct checks?

Reply:
Here's the relevant section from my initial comment on this PR.

> Both `PDO::errorInfo()` and `PDO::errorCode()` may now be called on an uninitialized PDO object, i.e. one without a defined `driver` member. This change is being done to permit use of both functions after `PDO::disconnect()` is called, and as a result of modifying the `PDO_CONSTRUCT_CHECK` macro to impose check on `is_closed`.

Both method implementations look to tolerate the absence of a defined `driver` member, with the exception of `methods->fetch_err()` invocation from within `errorInfo()`, where I've added the null check on `methods` member as safeguard.